client/src/elements/molecules/radio-button-group.css

```diff
@@ -2,16 +2,13 @@ radio-button-group {
   width: 100%;
   display: flex;
   flex-direction: row;
-  flex-wrap: wrap;
-  justify-content: flex-start;
 }
 
 .radio-button-group__option {
   position: relative;
-  margin-bottom: calc(var(--q-ui-space) / 2);
   flex-shrink: 0;
-  width: 36px;
-  height: 36px;
+  width: 110%;
```

Comment:
can you explain why this is `110%`? seems like magic at a first glance, then it's always good to explain or find a more straight forward way of solving the problem.

Reply:
Fixed in https://github.com/nzzdev/Q-editor/pull/119/commits/b846569e7c9a796af989ed87ae219d361f6b58fd